db/migrate/20190107145810_add_audit_id_to_adjustments.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuditIdToAdjustments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :adjustments, :audit_id, :integer
+  end
+end
```

Comment:
Invert this one -- 

Adjustments are also done independently of Audits. Reverse the association so that `Audit` has an `adjustment_id` field